stores/metadata.go

```diff
@@ -560,6 +554,38 @@ func (s *SQLStore) ContractSets(ctx context.Context) ([]string, error) {
 	return sets, err
 }
 
+func (s *SQLStore) PrunableData(ctx context.Context) (prunable int64, err error) {
+	err = s.db.
+		Raw(`
+SELECT IFNULL(SUM(prunable), 0)
+FROM (
+    SELECT ABS(c.size - COUNT(cs.db_sector_id) * ?) as prunable
```

Comment:
Why ABS? If the contract size is 100 and we have 1000 bytes of sector data for it then the prunable data is probably 0 and not 900.

Reply:
oh that's just a brain fart while trying to quickly fix a test